File: Microservices/Staffing.API/Models/SecurityGroup.cs
using System;

namespace Staffing.API.Models
{
    public class SecurityGroup
    {
        public Guid? Id { get; set; }
        public string GroupName { get; set; }
        public string RoleCodes { get; set; }
        public string FeatureCodes { get; set; }
        public string Notes { get; set; }
        public string OfficeRegionCodes { get; set; }
        public bool IsBossSystemUser { get; set; }
        public DateTime LastUpdated { get; set; }
        public string LastUpdatedBy { get; set; }
    }
}

File: AngularSPA/src/app/shared/interfaces/securityGroup.ts
export interface SecurityGroup {
  id: string;
  groupName: string;
  roleCodes: string;
  featureCodes: string;
  notes: string;
  lastUpdated: Date;
  lastUpdatedBy: string;
}

export interface SecurityGroupDetails extends SecurityGroup {
  roleNames: string;
  lastUpdatedByName: string;
}
